### PR TITLE
Move performance profiler to Permanent Popup

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerPopupComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    returnToLiveObservable: Observable<void>;
+    performanceCollector: PerformanceViewerCollector;
+}
+
+export const PerformanceViewerPopupComponent: React.FC<IPerformanceViewerPopupComponentProps> = (props: IPerformanceViewerPopupComponentProps) => {
+    const { scene, layoutObservable, returnToLiveObservable, performanceCollector } = props;
+
+    return (
+        <div id="performance-viewer">
+            <>
```
Comment (sebavan) [inline inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerPopupComponent.tsx:24]: why is the fragment needed if it is already enclosed in a div ?

File: inspector/src/components/popupComponent.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class PopupComponent extends React.Component<IPopupComponentProps, { isCo
             this._window.addEventListener("beforeunload", () => this._window && onClose(this._window));
             this._window.addEventListener("resize", () => {
                 if (this.props.onResize) {
-                    this.props.onResize();
+                    this.props.onResize(this._window!);
```
Comment (sebavan) [inline inspector/src/components/popupComponent.tsx:75]: you could check the _window in the previous if to prevent the use of ! which is always scary :-)

**Review by sebavan — COMMENTED** [review]
LGTM only a couple nits, will let @RaananW do the final pass.

File: inspector/src/components/popupComponent.tsx (hunk 2 above)
Comment (RaananW) [reply inspector/src/components/popupComponent.tsx:75]:
```suggestion
                    this._window && this.props.onResize(this._window);
```

File: inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerComponent.tsx [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
-import { CanvasGraphComponent } from "../../../graph/canvasGraphComponent";
 import { IPerfLayoutSize } from "../../../graph/graphSupportingTypes";
-import { PopupComponent } from "../../../popupComponent";
-import { PerformanceViewerSidebarComponent } from "./performanceViewerSidebarComponent";
 import { PerformanceViewerCollector } from "babylonjs/Misc/PerformanceViewer/performanceViewerCollector";
 import { PerfCollectionStrategy } from "babylonjs/Misc/PerformanceViewer/performanceViewerCollectionStrategies";
 import { Tools } from "babylonjs/Misc/tools";
 import "babylonjs/Misc/PerformanceViewer/performanceViewerSceneExtension";
-import { PerformancePlayheadButtonComponent } from "./performancePlayheadButtonComponent";
+import { Inspector } from "../../../..";
```
Comment (RaananW) [inline inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerComponent.tsx:13]: load directly from the file in which it is declared.

File: inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerPopupComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+
+import {Scene} from "babylonjs/scene";
```
Comment (RaananW) [inline inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerPopupComponent.tsx:3]: nit - formatting

**Review by RaananW — COMMENTED** [review]
Great!!

Just adding on top of seb's comments.

**Comment by carolhmj** [comment]
Thanks for the comments y'all, adressed them <3

**Review by RaananW — APPROVED** [review]

**Pull request merged**